docs-svelte-kit/src/lib/eslint/scripts/syntax.js

```diff
@@ -0,0 +1,296 @@
+export const language = {
```

Comment:
Based on the html sample of https://microsoft.github.io/monaco-editor/monarch.html